### ISSUE TITLE
Please check your S3 location is correct and is in the same region and try again.

### ISSUE THREAD

**Issue by robomotic** [airbnb/streamalert#848]
## Background
I am using the STABLE branch of streamalert as below:

> commit 557fb3f604661cdd9bd36486cccc8ce3a34bd1f1 (HEAD, tag: v2.0.0, tag: stable)

### Description
I am following all the steps for the terraform deployment.

### Steps to Reproduce
My settings are:
zone: us-east-1
Upon launching the terraform init via:

> ./manage configure prefix alert
>  ./manage.py terraform init

 I receive this error message at the end:

> Apply complete! Resources: 72 added, 0 changed, 0 destroyed.
> Traceback (most recent call last):
>   File "./manage.py", line 1690, in <module>
>     main()
>   File "./manage.py", line 1685, in main
>     cli_runner(options)
>   File "/home/ubuntu/streamalert/stream_alert_cli/runner.py", line 65, in cli_runner
>     terraform_handler(options, CONFIG)
>   File "/home/ubuntu/streamalert/stream_alert_cli/terraform/handler.py", line 68, in terraform_handler
>     _terraform_init(config)
>   File "/home/ubuntu/streamalert/stream_alert_cli/terraform/handler.py", line 129, in _terraform_init
>     create_table('alerts', alerts_bucket, config)
>   File "/home/ubuntu/streamalert/stream_alert_cli/athena/handler.py", line 198, in create_table
>     if athena_client.check_table_exists(sanitized_table_name):
>   File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 309, in check_table_exists
>     result = self.run_query_for_results('SHOW TABLES LIKE \'{}\';'.format(table_name))
>   File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 284, in run_query_for_results
>     execution_id = self._execute_and_wait(query)
>   File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 99, in _execute_and_wait
>     response = self._execute_query(query)
>   File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 130, in _execute_query
>     raise AthenaQueryExecutionError('Athena query failed:\n{}'.format(err))
> stream_alert.shared.athena.AthenaQueryExecutionError: Athena query failed:
> An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: The S3 location provided to save your query results is invalid. Please check your S3 location is correct and is in the same region and try again. If you continue to see the issue, contact customer support for further assistance.

## Desired Change

Please let me know how to fix this. Athena is available on us-east-1 so I am not sure what is this about.

**Comment by robomotic** [comment]
This is the output of:
`aws s3 ls | grep streamalert`

> 2018-11-27 10:24:16 alert.streamalert.athena-results
> 2018-11-27 10:21:54 alert.streamalert.s3-logging
> 2018-11-27 10:22:02 alert.streamalert.secrets
> 2018-11-27 10:22:02 alert.streamalert.terraform.state
> 2018-11-27 10:24:25 alert.streamalerts

**Comment by ryandeivert** [comment]
hi @robomotic - thank you for filing this! and for verifying the destination does in fact exist. if you don't mind trying something for me, can you run this to see if it's reproducible?

`
manage.py athena create-table --bucket alert.streamalerts --table-name alerts
`

If the table already exists, this will be non-destructive and will just tell you it already exists. Thanks!

**Comment by robomotic** [comment]
It seems producing the same error:

> ./manage.py athena create-table --bucket alert.streamalerts --table-name alerts

```
[INFO 2018-11-28 06:34:24,507 (StreamAlertCLI:47)]: Issues? Report here: https://github.com/airbnb/streamalert/issues
Traceback (most recent call last):
  File "./manage.py", line 1690, in <module>
    main()
  File "./manage.py", line 1685, in main
    cli_runner(options)
  File "/home/ubuntu/streamalert/stream_alert_cli/runner.py", line 71, in cli_runner
    athena_handler(options, CONFIG)
  File "/home/ubuntu/streamalert/stream_alert_cli/athena/handler.py", line 291, in athena_handler
    options.schema_override)
  File "/home/ubuntu/streamalert/stream_alert_cli/athena/handler.py", line 198, in create_table
    if athena_client.check_table_exists(sanitized_table_name):
  File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 309, in check_table_exists
    result = self.run_query_for_results('SHOW TABLES LIKE \'{}\';'.format(table_name))
  File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 284, in run_query_for_results
    execution_id = self._execute_and_wait(query)
  File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 99, in _execute_and_wait
    response = self._execute_query(query)
  File "/home/ubuntu/streamalert/stream_alert/shared/athena.py", line 130, in _execute_query
    raise AthenaQueryExecutionError('Athena query failed:\n{}'.format(err))
stream_alert.shared.athena.AthenaQueryExecutionError: Athena query failed:
An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: The S3 location provided to save your query results is invalid. Please check your S3 location is correct and is in the same region and try again. If you continue to see the issue, contact customer support for further assistance.

```

**Comment by ryandeivert** [comment]
hi @robomotic can you test this again by chance? I think this should be resolved with [this change](https://github.com/airbnb/streamalert/pull/849) in 2.1.1

**Comment by ryandeivert** [comment]
I believe this was resolved with #849 so am closing out until informed otherwise.
